addons/cni-cilium/helm-values

```diff
@@ -3,14 +3,14 @@ cni:
 
 hubble:
   relay:
-    enabled: false
+    enabled: true
   tls:
     auto:
       method: cronJob
   ui:
-    enabled: false
+    enabled: true
 
-kubeProxyReplacement: strict
+kubeProxyReplacement: true
```

Comment:
@kron4eg Yes, it fails otherwise, they changed it to a boolean in 1.16, before that it indeed used to be `strict`/`disabled`. What we test in `addons/cni-cilium/cilium-config-map.yaml` is what we have in our API, which still uses `strict`/`disabled`, and for which I created #3414